pkg/wireguard-tools/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM linuxkit/alpine:3744607156e6b67e3e7d083b15be9e7722215e73 as tools
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/main > /etc/apk/repositories && \
+    echo http://dl-cdn.alpinelinux.org/alpine/edge/testing >> /etc/apk/repositories && \
+    apk update && \
```

Comment:
Yes, it usually works and then it breaks... we should probably build from source. Can we just remove all of `pkg/wireguard-tools` for now, happy to build some real examples next week/tomorrow, lets just merge the kernel bits